java/ql/lib/semmle/code/java/security/UnsafeDeserializationQuery.qll

```diff
@@ -116,7 +114,7 @@ private class SafeKryo extends DataFlow2::Configuration {
   /**
    * Holds when a `KryoPool.borrow` method is called.
    */
-  private predicate stepKryoPoolBorrowMethod(DataFlow::Node node1, DataFlow::Node node2) {
+  additional predicate stepKryoPoolBorrowMethod(DataFlow::Node node1, DataFlow::Node node2) {
```

Comment:
Why have these been made `additional`? Shouldn't they just be `private` instead?